app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div id="exp-new" class="container-fluid col-sm-12 col-md-6 offset-md-3 col-lg-8 offset-lg-2 h-100">
+  <%= render partial: 'shared/navbar' %>
+  <%= render partial: 'shared/back_button' %>
+  <div class="container col-md-6 offset-md-3">
+    <h5 class="fw-bold" style="text-align: center; font-family: proxima-nova, sans-serif;">
+      New Expense for: </h5>
+    <h3 class="fw-bold" style="text-align: center; font-family: proxima-nova, sans-serif; margin-bottom: 30px;">
+      <%= @category.name %></h3>
+    <%= form_with(model: [@category, @expense], url: user_category_expenses_path(current_user,
+      @category)) do |form| %>
+      <table>
+        <tr>
+          <td><%= form.label :name %></td>
+          <td><%= form.text_field :name, class: "form-control" %></td>
+        </tr>
+        <tr>
+          <td><%= form.label :amount %></td>
+          <td><%= form.text_field :amount, class: "form-control", pattern: "[0-9]+([\.,][0-9]+)?" %></td>
+        </tr>
+      </table>
+      <% if can? :create, Expense %>
+        <%= form.submit "Create Expense", class: "btn btn-primary", style: "width: 100%; margin-top: 105px" %>
```

Comment:
Great work so far! However, there is still room for improvement. The current expenses form has two input fields for name and amount, but we need to add another field for categories. According to the project requirements, it is necessary to allow new expenses to be added to multiple categories, as shown in the ER diagram. At least one category must be added for each expense. You can use the `select` tag for that.
![image](https://github.com/jcmunav63/rails-budget-app/assets/55840999/ca13fc30-ec79-41d9-a3b6-ef1c28051923)
*